pkg/config/config.go

```diff
@@ -46,6 +45,21 @@ var DefaultConfig = Config{
 	HydraFilter:    false,
 }
 
+//importing cids from one tool to another
+var Reading_CIDs_From_File_Config = Config{
```

Comment:
I'm not sure what the `Reading_CIDs_From_File_Config` adds to the configuration, most of the fields  are duplicated. 

What about leaving the Default configuration like it is, and just changing the default values that each [cmd](https://github.com/cortze/ipfs-cid-hoarder/blob/4e3172a65de44da3ff0745b566c55951da0289a1/cmd/run_cmd.go#L45) applies when it is called.

You can actually assign directly: 
`DefaultConfig.CidFile` to `cids/cid-list.txt`
`DefaultConfig.CidSource` to any of the already existing `Source types` listed [above](https://github.com/cortze/ipfs-cid-hoarder/blob/165c2126bdf53337766375c97b13cff8fbb118a9/pkg/config/config.go#L15)

Reply:
Yeap doesn't add anything in particular I thought there would be some ready functionality for a hardcoded struct (I think it's a good idea to be added.)